src/rdkafka_topic.h

```diff
@@ -124,7 +124,7 @@ struct rd_kafka_topic_s {
 
         rwlock_t rkt_lock;
         rd_kafkap_str_t *rkt_topic;
-
+        rd_kafka_Uuid_t rkt_id;
```

Comment:
In this file it also needs to declare
```c
rd_kafka_topic_t *rd_kafka_topic_find_by_topic_id(rd_kafka_t *rk,
                                                  rd_kafka_Uuid_t topic_id);
``` 
Because it's used in fetcher.c